### PR TITLE
YulyaNyu_testToolsHeaderInToolsMenu

### PR DESCRIPTION
**Pull request by YulyaNyu**

### CODE REVIEW

File: src/test/java/school/redrover/GroupQaClimbersTest.java [hunk 1]
```diff
@@ -108,4 +108,17 @@ public void testAddProfileDescription() {
...
     }
+
+    @Test
+    public void testToolsHeaderInToolsMenu() {
+
+        getDriver().findElement(By.xpath("//a[@href = '/manage']")).click();
+
+        getDriver().findElement(By.xpath("//a[@href = 'configureTools']")).click();
+
+        WebElement toolsHeader = getDriver().findElement(By.cssSelector(".jenkins-app-bar__content >h1"));
```
Comment (ArturSabanadze) [inline src/test/java/school/redrover/GroupQaClimbersTest.java:119]: Are you gonna use the toolsHeader more than one time in your Test ?
If not then you could make your code more compactible/clear by replace line 119 and 120 with one line:

 final String actualVersion = getDriver().findElement(By.cssSelector(".jenkins-app-bar__content >h1")).getText();

Comment (YulyaNyu) [reply src/test/java/school/redrover/GroupQaClimbersTest.java]: ok, thank you, check please

**Review by ArturSabanadze — APPROVED** [review]

File: src/test/java/school/redrover/GroupQaClimbersTest.java (hunk 1 above)
Comment (StoyanaDi) [reply src/test/java/school/redrover/GroupQaClimbersTest.java]: Hi! I am interested why the variable String should be final? 
P.s I think that if is field in the class and used many time can be final, but variable in test that is used 1 time can be only string.

File: src/test/java/school/redrover/GroupQaClimbersTest.java [hunk 2]
```diff
@@ -108,4 +108,16 @@ public void testAddProfileDescription() {
...
     }
+
+    @Test
+    public void testToolsHeaderInToolsMenu() {
+
+        getDriver().findElement(By.xpath("//a[@href = '/manage']")).click();
+
+        getDriver().findElement(By.xpath("//a[@href = 'configureTools']")).click();
+
+        final String actualResult = getDriver().findElement(By.cssSelector(".jenkins-app-bar__content >h1")).getText();
```
Comment (StoyanaDi) [inline src/test/java/school/redrover/GroupQaClimbersTest.java:119]: I think that here final keyword is not necessary. 
P.s. if some var. is final must be uppercase using.

**Review by StoyanaDi — APPROVED** [review]

**Pull request merged**